src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const doc = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  const click = doc.addEventListener('click', (e) =>
+    resolve('First promise was resolved'));
+
+  setTimeout(() => {
+    if (!click) {
+      reject(new Error('First promise was rejected'));
+    }
```

Comment:
```suggestion
      reject(new Error('First promise was rejected'));
```
Не обов'язковий if addEventListener завжди повертає undefined відповідно цей if завжди спрацьовуватиме